common/src/main/java/com/datastrato/gravitino/config/ConfigConstants.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.config;
+
+public interface ConfigConstants {
+  String NOT_BLANK_ERROR_MSG = "The value can't be blank";
```

Comment:
I think using 'Missing config' can more clearly indicate the error.

Reply:
If you configure " ",  the validator will report error, too.